src/app/services/tmdb-service.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TmdbServiceService {
+
+  private endpoint = 'https://api.themoviedb.org/3';
+  private apiKey = '66681a441b89e003faa0833a4ac07f91';
```

Comment:
Api key mag nooit in je repository staan, deze graag verplaatsen naar een environment file